Lib/fontbakery/profiles/universal.py

```diff
@@ -330,22 +330,37 @@ def is_up_to_date(installed, latest):
     """,
     proposal = 'https://github.com/googlefonts/fontbakery/issues/2093'
 )
-def com_google_fonts_check_fontbakery_version():
+def com_google_fonts_check_fontbakery_version(font):
     """Do we have the latest version of FontBakery installed?"""
-    import json
     import requests
     import pip_api
 
-    pypi_data = requests.get('https://pypi.org/pypi/fontbakery/json')
-    latest = json.loads(pypi_data.content)["info"]["version"]
+    try:
+        response = requests.get('https://pypi.org/pypi/fontbakery/json')
+
+    except requests.exceptions.ConnectionError as err:
+        return SKIP, Message(
```

Comment:
A SKIP would be easy to silently ignore. We'd like to ensure latest Font Bakery is always being used, so a FAIL here would not let us miss the fact that the validation was not possible to be performed for whatever issue on the network. It would force us to re-run the check in order to make sure we've got a good result (or to figure out the check itself has some problem that needs to be fixed).

Reply:
I strongly disagree with changing this to FAIL because it would force FB users to always have a working internet connection whenever they run the Universal and other profiles.

Reply:
No, it wouldn't force anyone to be online. It would run just fine while offline. In that scenario, though, it would include a FAIL result indicating, out of an abundance of caution, that it was not possible to evaluate whether the version of Font Bakery being used is the latest available or not.

Fet me try to provide a more detailed explanation here. The reasoning is that newer versions of Font Bakery may include additional (or updated) quality-assurance criteria for which the font being checked might FAIL. So, a bland SKIP would be potentially hiding multiple FAILs.

A user who's working on a font family while offline will get a FAIL in this check and is expected to treat it as a reminder that a final Font Bakery run is still needed whenever the user is back online before giving the green light to safely publish a new font release. I think that's a reasonable scenario since the act of publishing a new font release would require access to the Internet anyway,

Reply:
I'll resolve this by changing it to FAIL here, and overriding it to SKIP in the Adobe Fonts profile.

Reply:
That's perfect! Thanks.

Reply:
Changes implemented in a9096c413a0122198a7895bb77c54ab3fd5b53e7 and e7b888ef37214f7f98de1d6c7835f6f1a94893bf